src/main/resources/model.ttl

```diff
@@ -200,3 +204,54 @@ rm:rejected-record-phase rdf:type owl:Class ;
         rdfs:label "rejected record phase"@en .
 
 ###  Generated by the OWL API (version 4.2.8.20170104-2310) https://github.com/owlcs/owlapi
+
+#################################################################
+#    New Roles
+#################################################################
+
+
+###  http://onto.fel.cvut.cz/ontologies/record-manager/complete-records
+rm:complete-records rdf:type owl:Class ;
```

Comment:
I guess it would be better if this is individual and other values as well.

Reply:
something like:
`rm:complete-records rdf:type owl:NamedIndividual, rm:role ;`

Reply:
maybe even call it rm:complete-records-role ?

Reply:
@blcham 

If I understand correctly I should write the following(just an example):

```
rm:role rdf:type owl:Class .
rm:group rdf:type owlClass.

rm:complete-records-role rdf:type owlNamedIndividual, rm:role ;
                       rdfs:label "Complete Records"@en .

```

rm:operator rdf:type owlNamedIndividual, rm:group;
                       has-role rm:complete-records-role.

Reply:
done